webui/react/src/pages/ExperimentDetails/ExperimentVisualization.tsx

```diff
@@ -372,6 +372,8 @@ const ExperimentVisualization: React.FC<Props> = ({ basePath, experiment }: Prop
         />
       </div>
     );
+  } else if (experiment.state === RunState.Error) {
```

Comment:
Experiments with multiple trials has nothing to show in Error state
For example, http://latest-main.determined.ai:8080/det/experiments/365/visualization/learning-curve